includes/Core/Assets/Assets.php

```diff
@@ -475,21 +475,11 @@ private function get_inline_data() {
 
 		if ( isset( $_GET['permaLink'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
 			$permalink = esc_url_raw( $_GET['permaLink'] );
+		} elseif ( is_front_page() ) {
+			$permalink = esc_url_raw( $site_url );
```

Comment:
I still fail to understand why we need the separate condition for the front page. We should show stats for individual posts, which the rest of the code already covers.